runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsConfig.java

```diff
@@ -53,6 +53,61 @@ default boolean rememberTab()
 		return true;
 	}
 
+	@ConfigItem(
```

Comment:
This should just work if bank tags plugin is enabled

Reply:
I don't think it should be on by default if bank tags is enabled, as it forces a filter on all items and all tags.

Reply:
It shouldn't always be searching price value.

Reply:
The original issue didn't specify how to do the filtering.

How do you think it should be activated then?
If you have to type the keyword in every time the search resets it's very cumbersome in my opinion. Rather, I let the user toggle it on and off in the settings.

Reply:
Through search tags of `ge` or `ha` would make more sense rather than always having a clunky filter on. 

Plus it's faster to enable it that way.

Reply:
How about a "price:" tag, and let the user toggle HA or GE values in the sidebar? I don't think these would be changed very much, and easier to just remember 1 price tag.

It'll use a similar method of checking if the user is searching by tag and also let the user filter by item name as well.

Reply:
why would you want that much complexity